app/grants/clr.py

```diff
@@ -647,9 +647,16 @@ def grants_transaction_validator(list_contributions):
     check_amount = lambda amount: int(amount[75:], 16) if len(amount) == 138 else print (f"{bcolors.FAIL}{bcolors.UNDERLINE} {index_transaction} txid: {transaction_tax[:10]} -> status: 0 False - amount was off by 0.001 {bcolors.ENDC}")
     check_token = lambda token_address: len(token_address) == 42
     check_contract = lambda token_address, abi : w3.eth.contract(token_address, abi=abi)
-    check_event_transfer =  lambda contract_address, search, txid : w3.eth.filter({ "address": contract_address, "topics": [search, txid]})
     get_decimals = lambda contract : int(contract.functions.decimals().call())
 
+    # check event filter
+    def check_event_transfer(contract_address, search, txid):
```

Comment:
for some reason this txn doesnt seem to show any transfers for this method https://etherscan.io/tx/0xd874b4fceb06b0611315394ec161c2ef874d6c6a9d68af99ff8570a08bd20510

Reply:
Possible fix: https://gist.github.com/inspireme6/361a5c00e8fc97c15301c4d127c6a5ba